### PR TITLE
Update and rename creating-apps-for-github-marketplace.md to empire-a…

### PR DESCRIPTION
**Pull request by empmenz**
…pps-marketplace.md

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by jace4411** [comment]
> …pps-marketplace.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by Ekkarat30 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Comment by jace4411** [comment]
> …pps-marketplace.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Comment by adamhayrettin11** [comment]
Aprovve

9 Oca 2021 Cmt 16:04 tarihinde jace4411 <notifications@github.com> şunu
yazdı:

**Comment by adamhayrettin11** [comment]
Lion heart

9 Oca 2021 Cmt 16:05 tarihinde jace4411 <notifications@github.com> şunu
yazdı:

**Comment by adamhayrettin11** [comment]
Victory gallop

9 Oca 2021 Cmt 16:05 tarihinde jace4411 <notifications@github.com> şunu
yazdı:

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by shivu0 — COMMENTED** [review]
Thanks

**Review by jace4411 — APPROVED** [review]

File: content/developers/github-marketplace/empire-apps-marketplace.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Creating apps for GitHub Marketplace
```
Comment (empmenz) [inline content/developers/github-marketplace/empire-apps-marketplace.md:2]: Empire Project

**Pull request closed without merging**